blocks/author-bio-block/features/author-bio/default.test.jsx

```diff
@@ -457,8 +447,8 @@ describe("Given the list of author(s) from the article", () => {
 
 		const wrapper = mount(<AuthorBio />);
 
-		const socialButtonsContainer = wrapper.find("section.socialButtons");
-		expect(socialButtonsContainer.children()).toHaveLength(0);
+		const socialLinks = wrapper.find({ "data-testid": "social-links" });
```

Comment:
not sure I follow what's happening here with data-testid. this is more of a convention with rtl as a last resort https://testing-library.com/docs/queries/bytestid/ 

I'd recommend maybe using .text() or finding by a classname

Reply:
Thanks jack, will work on this

Reply:
Sounds good!

Reply:
Fixed in [3b44189](https://github.com/WPMedia/arc-themes-blocks/pull/1474/commits/3b44189afea3a73af28c82ae7096bc81adffc3e1)

Reply:
this hasn't been addressed. We want to make sure data-testid is also removed in the jsx